libpod/container_internal.go

```diff
@@ -1166,6 +1167,18 @@ func (c *Container) start() error {
 
 	c.state.State = define.ContainerStateRunning
 
+	if c.config.SdNotifyMode != define.SdNotifyModeIgnore {
+		payload := fmt.Sprintf("MAINPID=%d\n", c.state.ConmonPID)
+		if c.config.SdNotifyMode == define.SdNotifyModeConmon {
+			payload += "READY=1\n"
```

Comment:
Apologies if this is a dup - I think github ate my earlier attempt to comment here.

Is there a reason to use hardcoded `READY=1` instead of `SdNotifyReady` from `go-systemd`?

Also, are you absolutely certain about the newline? It's been a long time since I played with sdnotify but IIRC there is no trailing newline. I've confirmed this just now by playing with `systemd-notify` and `socat`

Reply:
It works both ways - so I suppose it shouldn't be \n terminated then...  Resolved.